bluebottle/events/tests/test_api.py

```diff
@@ -582,7 +582,7 @@ def test_delete_by_other_user(self):
         self.assertEqual(data['errors'][0], "Transition is not available")
 
     def test_approve(self):
-        self.review_data['data']['attributes']['transition'] = 'approve'
+        self.review_data['data']['attributes']['transition'] = 'reject'
```

Comment:
Hmmm not sure about this. Shouldn't we test that you can or cannot use the approve transition?
Now at least the method name should change....

Reply:
I kind of see what you mean. Also test that a transition that does not exist correctly works.
I'll have a quick look.